### PR TITLE
feat: Add current date appointments and fix issue with displaying past and upcoming appointments

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary
This PR addresses two issues:

- Fixes the display of `Past` and `Upcoming` appointments in the UI. Presently, appointments with UTC dates are displayed incorrectly.
- Adds a `Today` tab to the content switcher that displays appointments for the present day. This information is useful in a clinical setup.

## Screenshots
Screenshot 2022-11-11 at 8 50 14 am

### CODE REVIEW

File: packages/esm-patient-appointments-app/src/appointments/appointments-base.component.tsx [hunk 1]
```diff
@@ -63,6 +65,21 @@ const AppointmentsBase: React.FC<AppointmentsBaseProps> = ({ patientUuid }) => {
...
+          if (contentSwitcherValue === AppointmentTypes.TODAY) {
+            if (appointmentsData.todaysAppointments?.length) {
+              return <AppointmentsTable patientAppointments={appointmentsData?.todaysAppointments} />;
+            }
+            return (
+              <Layer>
+                <Tile className={styles.tile}>
+                  <EmptyDataIllustration />
+                  <p className={styles.content}>
+                    {t('noCurrentAppointments', 'There are current appointments to display for this patient')}
```
Comment (makombe) [inline packages/esm-patient-appointments-app/src/appointments/appointments-base.component.tsx:77]:
```suggestion
                    {t('noCurrentAppointments', 'There are no current appointments to display for this patient')}
```

**Review by makombe — APPROVED** [review]
LGTM @donaldkibet thanks. Just a small change there to effect

File: packages/esm-patient-appointments-app/translations/en.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
   "discard": "Discard",
   "loading": "Loading",
   "location": "Location",
+  "noCurrentAppointments": "There are current appointments to display for this patient",
```
Comment (denniskigen) [inline packages/esm-patient-appointments-app/translations/en.json:18]:
```suggestion
  "noCurrentAppointments": "There are no appointments scheduled for today to display for this patient",
```

File: packages/esm-patient-appointments-app/src/appointments/appointments-base.component.tsx (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-appointments-app/src/appointments/appointments-base.component.tsx:77]: I'd suggest:

```
 "noCurrentAppointments": "There are no appointments scheduled for today to display for this patient",
```

**Review by denniskigen — COMMENTED** [review]
LGTM, thanks, @donaldkibet.

**Review by denniskigen — APPROVED** [review]

**Pull request merged**